src/client.c

```diff
@@ -768,10 +768,17 @@ handle_message (const char *message,
 
     jobj = json_tokener_parse(message);
     json_headers = find_object(jobj, "headers");
-    if (!json_headers) {
-        g_message("%s", message);
+
+    /* Every message that is not a JSON object is printed to STDOUT.
+       In a nutshell, these should be restraind own messages, NOT task,
+       NOT harness. */
```

Comment:
This doesn't read well.  Isn't restraintd the same as the harness?  Not sure the distinction you are trying to make.  You also have a typo s/restraind/restraintd/